File: das_backend/src/main/java/ch/sbb/backend/tenancy/infrastructure/config/TenantConfig.java
package ch.sbb.backend.tenancy.infrastructure.config;

import ch.sbb.backend.tenancy.domain.model.Tenant;
import java.util.List;
import lombok.Getter;
import lombok.Setter;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.boot.context.properties.EnableConfigurationProperties;
import org.springframework.context.annotation.Configuration;

@Setter
@Getter
@Configuration
@EnableConfigurationProperties
@ConfigurationProperties(prefix = "auth")
public class TenantConfig {

    private List<Tenant> tenants;

}

File: backend/src/main/kotlin/ch/sbb/backend/infrastructure/configuration/TenantConfig.kt
package ch.sbb.backend.infrastructure.configuration

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.context.annotation.Configuration

@Configuration
@EnableConfigurationProperties
@ConfigurationProperties("auth")
class TenantConfig {
    var tenants: List<Tenant> = ArrayList()
}
